recipes/cmake/3.x.x/conanfile.py

```diff
@@ -38,7 +38,7 @@ def config_options(self):
 
     def requirements(self):
         if self.options.with_openssl:
-            self.requires("openssl/1.1.1t")
+            self.requires("openssl/1.1.1w")
```

Comment:
Why don't you use version range like all other recipes?

Reply:
because versions of cmake go far back and the support for 3.x is not clear. Given those are not being built by cci, I'd rather go the safe way at first. It can still be made a version range later with the correct pivot.

Reply:
Could you test if it pass with openssl 3.x? It would avoid all these PR bumping versions of dependencies.

Reply:
I can't test this on macOS. The  build does not use OpenSSL.

Reply:
So why openssl is dragged irrespective of os? 🤔

Reply:
> Do a test build in CI with `self.requires("openssl/[>=1.1 <4]")` and update the recipe accordingly, perhaps?

Alternatively, just use the version range for the newest CMake versions and leave it to 1.1 for older.